cpp/src/strings/count_matches.cu

```diff
@@ -62,11 +63,14 @@ struct count_dispatch_fn {
 
   template <int stack_size>
   std::unique_ptr<column> operator()(column_device_view const& d_strings,
+                                     size_type output_size,
                                      rmm::cuda_stream_view stream,
                                      rmm::mr::device_memory_resource* mr)
   {
+    cudf_assert(output_size >= d_strings.size() and "Unexpected output size");
```

Comment:
Shouldn't this be a raw `assert`, not `cudf_assert`? This is host code, not device code, right?

Reply:
Yes, you are right.